llvm/lib/CASObjectFormats/NestedV1.cpp

```diff
@@ -445,12 +446,12 @@ SectionRef::create(const ObjectFileSchema &Schema, NameRef SectionName,
 
   // FIXME: Does 1 byte leave enough space for expansion? Probably, but
   // 4 bytes would be fine too.
-  B->Data.push_back(uint8_t(data::encodeProtectionFlags(Protections)));
+  B->Data.push_back(uint8_t(MemProt));
```

Comment:
Please do not drop use of `encodeProtectionFlags`. This is one of the few places where we're not directly encoding a `jitlink` enum, and eventually we want to be completely separate, so this is a step backward.

Reply:
(Instead, you should update `encodeProtectionFlags()` to take `jitlink::MemProt`)